examples/llm_inference/android/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <application
+        android:allowBackup="true"
+        android:dataExtractionRules="@xml/data_extraction_rules"
+        android:fullBackupContent="@xml/backup_rules"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:roundIcon="@mipmap/ic_launcher_round"
+        android:supportsRtl="true"
+        android:theme="@style/Theme.LLMInference"
+        tools:targetApi="31">
+        <activity
+            android:name=".MainActivity"
+            android:exported="true"
+            android:label="@string/app_name"
+            android:theme="@style/Theme.LLMInference">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
+        </activity>
+
+        <!-- Required to initialize the LlmInference -->
+        <uses-native-library
+            android:name="libOpenCL.so"
+            android:required="false"/>
+        <uses-native-library android:name="libOpenCL-car.so" android:required="false"/>
```

Comment:
Why are there 3 different versions of libOpenCL here?

Reply:
I'm not sure, tbh. I was just copying from the work-in-progress docs. From my testing, it seems like the app only breaks if `libOpenCL.so` is not included.

@schmidt-sebastian do you know why (or if) we need `libOpenCL-car.so` and `libOpenCL-pixel.so` here?

Reply:
I don't

Reply:
If not, it'll crashed in some devices (IIRC Pixel). We did the same thing with Image Generator.
https://github.com/googlesamples/mediapipe/blob/0fc6e2b809b13b7cb1de1792288409624f5f6007/examples/image_generation/android/app/src/main/AndroidManifest.xml#L14

Why we need that many library? -> better ask Pixel team :)